lib/parser.js

```diff
@@ -389,9 +390,37 @@ const args = [
     help: 'Disable additional security checks, so it is possible to use some advanced features, provided ' +
           'by drivers supporting this option. Only enable it if all the ' +
           'clients are in the trusted network and it\'s not the case if a client could potentially ' +
-          'break out of the session sandbox.',
+          'break out of the session sandbox. Specific features can be overridden by ' +
+          'using the --deny-insecure flag',
     nargs: 0
   }],
+
+  [['--allow-insecure'], {
+    dest: 'allowInsecure',
+    defaultValue: [],
+    type: parseSecurityFeatures,
+    required: false,
+    example: 'execute-webdriver,adb-shell',
```

Comment:
I'm still not sure how you are going to transform existing commands to these abbreviations. Or there will be another separate mapping?

Reply:
Me, too.
Maybe, should we create a mapping file...?
execute-webdriver is probably easy since the command is a new command in routes.js. We can filter it with the command, _executeDriverScript_, for example.
The adb shell is defined in a part of mobile commands. How to map them... 🤔 https://developer.android.com/studio/command-line/adb#shellcommands

- adb-shell # all adb shell
-  _adb shell_ and one argument
    - e.g. adb-shell-pm (for [this](https://developer.android.com/studio/command-line/adb#pm)), adb-shell-pm, adb-shell-dpm, adb-shell-screencap, adb-shell-screenrecord...?

Reply:
I don't think we need any mapping. Different drivers can define their own feature names. All that matters is that the drivers (a) actually check whether a feature is enabled if that feature is insecure, and (b) document the feature names so people know what to use. I'll also create a new doc in the main docs area to list known feature names for the officially supported drivers.

Reply:
I could imagine with https://github.com/appium/appium-base-driver/pull/328/files#diff-69c68e3397852a86397892936887db6aR258
What about addressing usage? of `isFeatureEnabled` for developers in docs/en/writing-running-appium/security.md later? It helps for other driver developers to get the security feature in their drivers